ansible/roles/alerts_box/files/prometheus_rules/validators-ejector.rule.yml

```diff
@@ -2,31 +2,59 @@ groups:
   - name: Validators-Ejector
     rules:
       - alert: ValidatorsEjectorExitMessages
-        expr: validator_ejector_exit_messages > 1
+        expr: validator_ejector_exit_messages{valid="false"}  > 0
         labels:
           severity: warning
           service: validator_ejector
         annotations:
-          summary: Validators-Ejector exit messages amount too high.
-          description: "{{ $value }} exit messages in Validators-Ejector queue"
+          summary: Validator-Ejector loaded invalid exit messages.
+          description: "{{ $value }} invalid exit messages were loaded by Validator-Ejector. Please check app logs and messages format, signature and validator index."
           runbook: "https://hackmd.io/@lido/Byue6SQxh"
 
-      - alert: ValidatorEjectorJobErrors
-        expr: validator_ejector_job_duration_seconds_bucket{le="+Inf",result="error"} > 1
+      - alert: ValidatorEjectorJobDurationSecondsCount
+        expr: validator_ejector_job_duration_seconds_count{result="error"} >= 3
+        for: 1h
```

Comment:
Is it a right metric to use? 
As I understand that expression will catch if any failed job (`result=error`) will lasts longer than 3 seconds for the last 1 hour.